src/insn_builder.rs

```diff
@@ -0,0 +1,1389 @@
+// Copyright 2017 <alex.dukhno@icloud.com>
```

Comment:
Maybe with your name in addition to the email?

    // Copyright 2017 Alex Dukhno <alex.dukhno@icloud.com>